src/Dapr.Actors/DaprHttpInteractor.cs

```diff
@@ -78,7 +78,7 @@ HttpRequestMessage RequestFunc()
                 return request;
             }
 
-            return this.SendAsync(RequestFunc, relativeUrl, cancellationToken);
+            await this.SendAsync(RequestFunc, relativeUrl, cancellationToken);
```

Comment:
This was the fix for the unit test determinism issue. We should ultimately change how we're doing this testing so it's less fragile, but this should at least make the tests pass.